wurst/_handles/Item.wurst

```diff
@@ -122,3 +126,24 @@ public function item.getIconPath() returns string
 
 public function item.setIconPath(string path)
 	BlzSetItemIconPath(this, path)
+
+// Extensions
+/** Updates the item's name, tooltip, extendedtooltip, description or icon for the specific player.
+	If p == null, the item is updated for all players */
+public function item.updateItem(player p, string itemName, string itemTooltip, string itemExtendedTooltip, string itemDesc, string itemIcon) returns bool
```

Comment:
IMHO, this shouldn't be local to player, and then maybe have a 'ForPlayer' counterpart.

Reply:
why not localplayer?

Reply:
Guess to not include Player package, cause _handle folder is already pretty much interconnected, and including it here would create loop dependency.
But again, this does not need to be player specific.

Reply:
Yeah that is true ofc.
We used it for only localplayer changes thats why we had it this way, guess the localplayer should be removed, and localplayer condition be checked on usage
Have the same for Frameapi btw...

Reply:
I don't really get the point.
The tooltip change can occur on both local player only, or for all players, and this functions covers both.
But I don't really mind having two separate functions for this.